installing/installing-preparing.adoc

```diff
@@ -40,7 +40,7 @@ endif::openshift-origin[]
 
 You can deploy an {product-title} 4 cluster to both on-premise hardware and to cloud hosting services, but all of the machines in a cluster must be in the same data center or cloud hosting service.
 
-If you want to use {product-title} but do not want to manage the cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, IBM Cloud VPC, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page.
+If you want to use {product-title} but do not want to manage the cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, IBM Cloud VPC, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. If you install an {product-title} cluster with a cloud virtual machine as a virtual bare metal, the corresponding cloud-based storage is not supported.
```

Comment:
@snarayan-redhat I am not sure what _as a virtual bare metal_ means.

Also, I'm not an exper here, but I wonder if we should tell people, as the Jira suggests, what they need to do to if they want cloud-based storage. 

> the Dynamic Provisioning will only work if we did Azure cloud provider (IPI/UPI) installation and not the baremetal/UPI installation.

Reply:
@mburke5678 I was on the same page as you. This was suggested and approved by SME and also double confirmed with QE on the term's validity.  Also regarding the ticket, the context of the issue went beyond dynamic provisioning. The bug fix started with the Jira's context, and as per SME's comment, it applied to OCP in whole.